src/components/NewWorkspaceModal.js

```diff
@@ -30,6 +30,35 @@ const constraints = {
   }
 }
 
+const allRegions = [
```

Comment:
This list probably belongs in https://github.com/DataBiosphere/terra-ui/blob/dev/src/components/region-common.js too